django/applications/catmaid/control/landmarks.py

```diff
@@ -1465,8 +1460,8 @@ def get(self, request:Request, project_id, landmarkgroup_id) -> Response:
         group_ids = None
         cursor = connection.cursor()
         if relation.isreciprocal:
-            cursor.execute("""
-                -- This assumes a reciprocal relation, direction is ignored.
+            # This assumes a reciprocal relation, direction is ignored.
```

Comment:
Felt weird to send comments to the SQL engine. The codebase does this several places.

Reply:
I know, and that's okay, but in some more complex queries it helps to have comments in specific locations of the SQL. I doubt this has a significant performance impact.